lua/telekasten.lua

```diff
@@ -2054,6 +2070,7 @@ local function on_create(opts, title)
             map("n", "<esc>", picker_actions.close(opts))
             return true
         end,
+        sort = M.Cfg.sort,
```

Comment:
Is it really necessary here? This picker is for template selection, so it does not really make sense to have it by modified (imo).

Reply:
I put it on, but you're right, I don't seem to need that, so I'm going to delete it.